lib/beaker/shared/host_manager.rb

```diff
@@ -102,7 +102,7 @@ def run_block_on hosts = [], filter = nil, opts = {}, &block
         end
         if block_hosts.is_a? Array
           if block_hosts.length > 0
-            if opts[:run_in_parallel]
+            if opts[:run_in_parallel] == true
```

Comment:
@nwolfe My original PR had 2 different options, one called install_in_parallel which was a global option, and one called run_in_parallel which was an option that could be passed into run_block_on.  Based on feedback, I changed the global install_in_parallel option to run_in_parallel with an array of what 'things' to run in parallel.  The 2 current options are configure and install.  In retrospect, naming the global option the same thing as the run_block_on option is confusing and can possibly have collisions like this...